test_colab.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+boi="# begin of installation"
+eoi="# end of installation"
+for colab_file in ./unified_planning/notebook/*.ipynb ; do
+    file_name="${colab_file%.ipynb}"
+    jupyter nbconvert --to python ${colab_file}
+    python_file="${file_name}.py"
```

Comment:
Just to be sure that we convert to the right file :)
```suggestion
    python_file="${file_name}.py"
    jupyter nbconvert --to python ${colab_file} --output ${python_file}
```